py/newsletter.py

```diff
@@ -16,7 +16,7 @@
 greet_title = 'HAPPY WEEK * EVERYONE'
 greet_sub = 'Here are some things to get excited about!'
 
-def generate(sections, greet_title=greet_title, greet_sub=greet_sub):
```

Comment:
Can you revert this back so the parser doesn't break?

Reply:
Whoops I missed that thanks! Reverted!